pdfannots/printer/jsonl.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import json
+import typing
+
+from . import DictBasedPrinter
+from ..types import Document
+
+
```

Comment:
I'd appreciate a docstring or header comment here explaining the jsonl format (e.g. the Wikipedia link).